extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -79,8 +106,13 @@ export class BindValidator {
       rules = this.rulesForTagAndProperty_(tag, rules.alternativeName);
     }
 
-    // If there are no rules governing this binding, return true.
-    if (!rules) {
+    // If binding to (tag, property) is not allowed, return false.
+    if (rules === undefined) {
```

Comment:
It did at one point, but it causes a duplicate call to `rulesForTagAndProperty_`. This is slightly more efficient.